frontends/election-manager/src/utils/election.ts

```diff
@@ -59,6 +59,29 @@ export function getBallotStylesData(election: Election): BallotStyleData[] {
   );
 }
 
+const superBallotStyleId = 'vx-super-ballot';
+const superBallotStylePrecinctId = 'vx-all-precincts';
```

Comment:
Worth preventing ballot style and precinct ids from starting with `vx-` using zod schemas?

Reply:
Great call - didn't know zod schemas could do that! Opened a ticket: https://github.com/votingworks/vxsuite/issues/2385